docs/src/concepts.md

```diff
@@ -0,0 +1,76 @@
+# Concepts
+
+## Facial Motion Tracking
+
+Studying the inner workings of the brain requires understanding the relationship between
+neural activity and the facial activity. Motions in the facial features (e.g. whisker,
+eye/pupil, etc) is one type of behavioral activity that can be measured by Facemap.
+
+Motions in the face at the whisker or nose Facial motion Pose estimation is a computer
+vision method to track the position, and thereby behavior, of the subject over the
+course of an experiment, which can then be paired with neuronal recordings to answer
+scientific questions about the brain.
+
+Facemap allows running SVD on a region-of-interest either in the movie itself or in the
+motion movie (difference of frames). The region of interest Movie SVD: Frames itself.
+Motion SVD: Difference of frames.
+
+ROIs: User can select a rectangular, or ellipsoid shape.
+
+Returns PCA components that give distinct facial features.
+
+## Caveats
+
+The computation time and the memory usage can be pressing for large region of interests.
```

Comment:
```suggestion
```

Reply:
@tdincer @CBroz1 I suggest removing since it is not clear how this statement fits within this page.